rapthor/process.py

```diff
@@ -77,7 +77,37 @@ def run(parset_file, logging_level='info'):
                 if has_diverged:
                     log.warning("Selfcal has diverged (ratio of current image noise "
                                 "to previous value is > {})".format(field.divergence_ratio))
-                log.info("Stopping at iteration {0} of {1}".format(index+1, len(strategy_steps)))
+                log.info("Stopping selfcal at iteration {0} of {1}".format(index+1, len(strategy_steps)))
                 break
 
+    # Run with the final data fraction if needed
+    if parset['final_data_fraction'] != parset['data_fraction']:
```

Comment:
This is comparing floats isn't it. Maybe use `np.isclose` or something similar.

Reply:
Good idea. I updated it to use `np.isclose`.